### DISCUSSION TITLE
Unwanted comment style transformation

### DISCUSSION THREAD

**Discussion by otryshko** [mirumee/ariadne#705] [Q&A] [answered]
Hi,

I added a number of comments to my .graphql schema, using triple quote notation. 
```
type Query {
    """
    Foo description
    """
    foo: Foo
}
```
When I try to download the schema from playground, those comments show up as '# <comment>' in the downloaded sdl.

Is there a way to preserve comment style? I need to post-process the sdl file downloaded from playground, and code down the flow expects triple-quoted comments.

**Comment by rafalp** [comment, marked as answer]
AFAIR Information about comment syntax used is not preserved for schema object, so there's no way for Playground to know how to format those strings.
